### PR TITLE
Remove POD_ADDRESS_RANGE

### LINKED ISSUE #1258: Deprecate (and remove) POD_ADDRESS_RANGE
As discussed in #1209, POD_ADDRESS_RANGE should be removed and SUBNET should be used instead.
Tasks:
- remove all occurences of POD_ADDRESS_RANGE from containers
- update the docs
- write a changelog (breaking change!)

### PR DESCRIPTION
**Pull request by DjVinnii**
## What type of PR?

Removal

## What does this PR do?

As discussed in #1209 `POD_ADDRESS_RANGE` should be removed in favor of  `SUBNET`. This PR removes the few references that are still left.

### Related issue(s)
- closes #1258

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [x] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

File: core/admin/mailu/configuration.py [hunk 1]
```diff
@@ -87,8 +87,7 @@
     'HOST_REDIS': 'redis',
     'HOST_FRONT': 'front',
     'SUBNET': '192.168.203.0/24',
-    'SUBNET6': None,
```
Comment (nextgens) [inline core/admin/mailu/configuration.py:90]: hmm, SUBNET6 is still used; this must be a typo

**Review by nextgens — CHANGES_REQUESTED** [review]

File: core/admin/mailu/configuration.py (hunk 1 above)
Comment (DjVinnii) [reply core/admin/mailu/configuration.py:90]: I removed the comma after `SUBNET6`, right?

Comment (nextgens) [reply core/admin/mailu/configuration.py:90]: doh, sorry. I need new specs!

**Review by Diman0 — APPROVED** [review]
lgtm

**Pull request merged**